fmatch/matcher.py

```diff
@@ -68,7 +67,11 @@ def query_index(self, index: str, search: Search) -> Response:
         return search.execute()
 
     def get_uuid_by_metadata(
-        self, meta: Dict[str, Any], index: str = None, lookback_date: datetime = None
+        self,
+        meta: Dict[str, Any],
+        index: str = None,
+        lookback_date: datetime = None,
```

Comment:
can we just add a doc string here, saying that lookback period is applied first and then lookback size on top of the output data?